app/controllers/groups_controller.rb

```diff
@@ -251,7 +251,7 @@ def members
       Discourse.deprecate(":desc is deprecated please use :asc instead", output_in_test: true, drop_from: '2.9.0')
       dir = (params[:desc] && params[:desc].present?) ? 'DESC' : 'ASC'
     end
-    order = ""
+    order = "NOT group_users.owner"
```

Comment:
I didn't confirm this but something feels off here. This variable is used for the following code below: 

```
    owners = users
      .order(order)
      .order(username_lower: dir)
      .where('group_users.owner')
      .includes(:primary_group)
 ```
 
 The ordering does not make sense because we have a filter for `where('group_users.owner')`. I think this order needs to be restricted to just our query for members of the group?

Reply:
In this case it is a no op and I did not want to further complicate the code.